clusterloader/framework/util.go

```diff
@@ -80,3 +113,86 @@ func CreatePods(f *framework.Framework, appName string, ns string, labels map[st
 		}
 	}
 }
+
+func CreateRC(f *framework.Framework, namespace, name, imageName string, label map[string]string, replicas int) error {
+	nodeSelector := fields.OneTermEqualSelector("metadata.name", name)
+	list, _ := f.ClientSet.Core().ReplicationControllers(namespace).List(metav1.ListOptions{FieldSelector: nodeSelector.String()})
```

Comment:
I don't understand this line - "FieldSelector: nodeSelector.String()" looks pretty bad to me.